bindings/python/QuinticSpline.cpp

```diff
@@ -0,0 +1,31 @@
+/**
+ * @file QuinticSpline.cpp
+ * @authors Giulio Romualdi, Diego Ferigo
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#include <pybind11/eigen.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "BipedalLocomotion/Planners/QuinticSpline.h"
+#include "bipedal_locomotion.h"
+
+void BipedalLocomotion::bindings::CreateQuinticSpline(pybind11::module& module)
+{
+    namespace py = ::pybind11;
+    using namespace BipedalLocomotion::Planners;
+
+    py::class_<QuinticSpline>(module, "QuinticSpline")
+        .def(py::init())
+        .def("set_knots", &QuinticSpline::setKnots)
+        .def("set_initial_conditions", &QuinticSpline::setInitialConditions)
+        .def("set_final_conditions", &QuinticSpline::setFinalConditions)
+        .def("evaluate_point",
+             py::overload_cast<const double&,
+                               Eigen::Ref<Eigen::VectorXd>,
+                               Eigen::Ref<Eigen::VectorXd>,
+                               Eigen::Ref<Eigen::VectorXd>>(
+                 &QuinticSpline::evaluatePoint));
```

Comment:
Probably here we can change the python signature to something like
```python
pos, vel, acc = spline.evaluate_point(t)
```
I think it can be easily done with a lambda function, right?

Reply:
I didn't edit the implementation of this class after you propose it in the previous pull request (now that I see it, I just used the `py::overload_cast` instead of the ugly pre-C++14 syntax for overloads). I'll check if it is possible returning a `std::tuple<double, double, double>`.

Reply:
Done in 69fa094. Just to understand, why 4 elements?

Reply:
I've just realized that the dimension of `pos` is not always 4. It depends on the size of the knots that should be equal to the size of the initial/final velocity/acceleration.

As far as I remember there is no way to retrieve the size of the points, we may think to add a method. in this case, the code may become
```cpp
Eigen::VectorXd pos(qs.size());
...
```

Reply:
How do you suggest to proceed?

Reply:
I think you can revert the previous implementation and that I can figure out how to change it. 
I can do it just after this PR got merged.

Reply:
I've rebased this PR to remove 69fa094.